File: Solved/09498/09498.cpp
// Problem No.: 09498
// Solver:      Jinmin Goh
// Date:        191120
// URL: https://www.acmicpc.net/problem/9498

#include <iostream>

using namespace std;

int main()
{
    int a;
    
    cin >> a;
    if(a >= 90)
        cout << "A";
    else if(a >= 80)
        cout << "B";
    else if(a >= 70)
        cout << "C";
    else if(a >= 60)
        cout << "D";
    else
        cout << "F";

    return 0;
}

File: Solved/09498/09498.py
# Problem No.: 09498
# Solver:      Jinmin Goh
# Date:        191120
# URL: https://www.acmicpc.net/problem/9498


a = input()

a = int(a)

if a >= 90:
    print("A")
elif a >= 80:
    print("B")
elif a >= 70:
    print("C")
elif a >= 60:
    print("D")
else:
    print("F")